### ISSUE TITLE
Remove folding markers

### ISSUE THREAD

**Issue by aeschli** [redhat-developer/vscode-java#369]
The folding markers should be removed from language-configuration.java as they are already provided by the built-in Java extension.

Also they have a bug as they match 
```
    System.out.println("<editor-fold>");
    System.out.println("Hello, world");
    System.out.println("</editor-fold>");
```
Same bug is in the built-in Java extension, but fixed in 1.19

**Comment by fbricon** [comment]
I'm happy to remove those once vscode 1.19 is out

**Comment by aeschli** [comment]
But better fix it then...

**Comment by fbricon** [comment]
The bug has been fixed in master by copying vscode's patterns. 
I'll keep this issue opened as a reminder to remove those patterns after the end of november release. vscode-java 0.16.0 will be released after vscode 1.19.
